src/App.tsx

```diff
@@ -1,11 +1,42 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { Fragment } from 'react';
+import { StyleSheet, SafeAreaView, View, AppRegistry } from 'react-native';
+import Amplify from 'aws-amplify';
+import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
+import awsconfig from './aws-exports';
+
+import AuthContainer from './components/auth-container';
+import SplashScreen from './components/splash-screen';
+
+const ApolloURI = 'localhost:4000/graphql';
```

Comment:
What I mean by an environmental variable is that you define the variable in an `.env` file, and you retrieve the variable ```const APOLLO_URI = `${process.env['APP_URI'] || 'localhost:4000'}/graphql`;```

Reply:
We're working on this one. Based on the [Expo docs](https://docs.expo.io/guides/environment-variables/), it seems we need to do some extra work to access environment variables. [This article](https://alxmrtnz.com/thoughts/2019/03/12/environment-variables-and-workflow-in-expo.html) and [this one too](https://medium.com/swlh/how-to-properly-use-environment-variables-in-an-expo-react-native-app-7ab852590b30) propose solutions to access environment variables in Expo.